R/demo-crowding.R

```diff
@@ -1,23 +1,29 @@
 #' Access to Crowding Data
 #' 
-#' This function returns a data frame from Crowding in State Cancer Profiles
+#' This function returns a data frame about crowding demographics from State 
+#' Cancer Profiles.
 #'
 #' @param area A state/territory abbreviation or USA.
-#' @param areatype Either "county", "hsa" (Health service area), or "state"
-#' @param crowding "household with >1 person per room"
-#' @param race One of the following values:
-#'             "All Races (includes Hispanic)",
-#'             "white (includes hispanic)",
-#'             "white non-hispanic",
-#'             "black",
-#'             "amer. indian/alaskan native (includes hispanic)",
-#'             "asian or pacific islander (includes hispanic)",
-#'             "hispanic (any race)"
+#' @param areatype Either `"county"`, `"hsa"` (Health Service Area), or
+#' `"state"`.
+#' @param crowding The only permissible value is 
+#' `"household with >1 person per room"`.
+#' @param race 
+#' One of the following values:
+#' - "All Races (includes Hispanic)"
+#' - "White (includes Hispanic)"
+#' - "White Non-Hispanic"
+#' - "Black"
+#' - "Amer. Indian/Alaskan Native (includes Hispanic)"
+#' - "Asian or Pacific Islander (includes Hispanic)"
+#' - "Hispanic (Any Race)"
```

Comment:
For more than three choices, these lists are handy and user friendly. Take a look at the rendered docs to see how this formatting comes out.